### PR TITLE
graph-links: restore previous scheme, cleanups

### PR DESCRIPTION
**Pull request by sevein**
I merged #13 too fast. This is a continuation for @Hwesta.

### CODE REVIEW

**Comment by Hwesta** [comment]
Update the README with the new graph-links information please

File: tools/graph-links [hunk 1]
```diff
@@ -35,10 +36,23 @@ databaseInterface.printSQL = False
...
+    '19c94543-14cb-4158-986b-1d2b55723cd8': 'reject sip option',
+}
+
+COLORS = {
+    'node_chain_link': 'black',
+    'node_chain_link_start': 'darkgoldenrod',
+    'edge_exit_codes_ok': 'green',
+    'edge_exit_codes_err': 'red',
+    'edge_user_selections': 'green',
+    'edge_watched_directories': 'yellow',
```
Comment (Hwesta) [inline tools/graph-links]: User selection edge is the same color as exit code OK.  Also, the yellow is often hard to read against a white background.  While you're changing things, maybe fix this? Black might also work for exit code OK.

File: tools/graph-links [hunk 2]
```diff
@@ -95,6 +109,18 @@ TASK_TYPES = {
 logging.basicConfig(format='%(message)s', stream=sys.stdout)
 logger = logging.getLogger()
 
+config = ConfigParser.ConfigParser()
+config.read('/etc/archivematica/MCPClient/archivematicaClientModules')
+
+
+def get_script_name(name):
+    if not isinstance(name, unicode):
```
Comment (Hwesta) [inline tools/graph-links]: Tip: Better to check for `isinstance(name, basestring)` http://stackoverflow.com/questions/4843173/how-to-check-if-type-of-a-variable-is-string-in-python

**Comment by sevein** [comment]
Ok, done. Thanks for the feedback.

**Comment by Hwesta** [comment]
:+1: New graph-links is much nicer!

**Pull request closed without merging**